src/views/wallet/UserAssetDetailsScreen.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import { WalletAssetDetailsRouteParams } from '../../routes'
+import TransactionsTable, { UserTransactionTypes } from '../../components/wallet/UserTransactionsTable'
+import DynamicCoin from '../../components/shared/CoinIcons/DynamicCoin'
+import { shortSymbol } from '../../helpers/tokenHelpers'
+import { Row, Col, Typography, Divider, Button, Card } from 'antd'
+
+const { Title } = Typography
+
+type UserAssetType = {
```

Comment:
Let's extract this `type UserAssetType` into `wallet/types` or somewhere, similar to `UserTransactionTypes`.